itou/metabase/management/test_populate_metabase_emplois.py

```diff
@@ -593,6 +593,48 @@ def test_populate_evaluated_criteria():
         ]
 
 
+@freeze_time("2023-02-02")
+@pytest.mark.django_db(transaction=True)
+@pytest.mark.usefixtures("metabase")
+def test_populate_users_exclude_job_seekers():
+    """
+    Job seeker personal data (email...) should never ever ever ever ever ever end up in Metabase.
```

Comment:
I think we forgot to say job seeker personal data should never end up in Metabase. Ever. Ever ever. Ever ever ever.